plugin/proxy/grpc_test.go

```diff
@@ -59,6 +64,97 @@ func TestStartupShutdown(t *testing.T) {
 	}
 }
 
+func TestRunAQuery(t *testing.T) {
+	grpclog.SetLogger(discard{})
+
+	upstream := &staticUpstream{
+		from: ".",
+		HealthCheck: healthcheck.HealthCheck{
```

Comment:
forgot, but is healthcheck need for this test?

Reply:
no, only the pool of hosts.